consensus_test.go

```diff
@@ -11,90 +15,13 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
-func TestTransition_ValidateState_Prepare(t *testing.T) {
```

Comment:
Have you removed this tests?

Reply:
`TestTransition_ValidateState_Prepare` is renamed to `TestTransition_ValidateState_CommitState` and slightly changed in order to satisfy conditions to move from Validate to Commit state.

On the other hand, `TestTransition_ValidateState_CommitFastTrack` is removed completely, since `TestTransition_ValidateState_CommitState` already tests `ValidateState` -> `CommitState` state transition.